README.md

```diff
@@ -1,116 +1,134 @@
+without the need to be trained in ontology curation or querying. Ontology-
+driven standards benefit from features of open-source published OWL 2.0 
+ontologies such as globally unique identifiers for terms, multilingual label 
+and definition functionality, and logical validation and reasoning over 
+controlled vocabularies. Such a specificatio#n can be designed to satisfy the 
+requirements of an environmental pH measurement, or a person's age, or a more 
+structured entity like a contact address, or a genomic sequence repository 
+submission for example. 
+
+* View details of our overal project aim, description, and implementation on 
+  the [Wiki](https://github.com/GenEpiO/geem/wiki/) site.
+* A working prototype of the portal is available at <http://genepio.org/geem>. 
+  This version doesn't currently incorporate the Django framework yet.
 
 ___________
 
-## Creating a Development Server
+## Deployment
 
-### Using Docker
-
-1. Build the GEEM docker image:
+Clone this repository.
 
 ```bash
-$ docker build .
+$ git clone https://github.com/GenEpiO/geem.git
 ```
 
-2. Set up the database:
+### Setting up social authentication
 
-```bash
-$ docker-compose run web python /code/manage.py makemigrations --noinput
-$ docker-compose run web python /code/manage.py migrate --noinput
-```
+To run a local server, create the file `.envs/.local/.auth`. To run a 
+production server, create the file `.envs/.production/.auth`. Neither of these 
+files are tracked by Github.
 
-3. Create your admin user:
+Templates of both files are found at `.envs/.local/.auth.template` and 
+`.envs/.production/.auth.template`.
 
-```bash
-$ docker-compose run web python /code/manage.py createsuperuser
-```
+#### SOCIAL_AUTH_GITHUB_KEY and SOCIAL_AUTH_GITHUB_SECRET
 
-4. Load initial data:
+[Register an Oauth2 application on Github.][1] If you are running a local 
+server, enter `http://localhost:8000/index.html` as Homepage URL, and 
+`http://localhost:8000/complete/github/` as Authorization callback URL. If 
+you are running a production server, enter 
+`http://localhost:8888/geem/index.html` as Homepage URL, and 
+`http://localhost:8888/geem/complete/github/` as Authorization callback URL.
 
-```bash
-$ docker-compose run web python /code/manage.py loaddata shared_packages
-```
+Once the application is created, enter the Client ID and Client Secret into 
+your `.auth` file as `SOCIAL_AUTH_GITHUB_KEY` and `SOCIAL_AUTH_GITHUB_SECRET` 
+respectively.
 
-5. Run the development server:
+#### SOCIAL_AUTH_GOOGLE_OAUTH2_KEY and SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET
 
-```bash
-$ docker-compose up
-```
+[Register an Oauth2 application on the Google Developers Console.][2] Select 
+_Web Application_ as the Application type. If you are running a local server, 
+enter `http://localhost:8000/complete/google-oauth2/` as a value for Authorized 
+redirect URIs. If you are running a production server, enter 
+`http://localhost:8888/geem/complete/google-oauth2/`.
```

Comment:
And here too. I recall NGIX port was changed to 80?

Reply:
[See here.](https://github.com/GenEpiO/geem/pull/45/files#r347107427)